airflow/providers/common/sql/operators/sql.py

```diff
@@ -190,11 +189,17 @@ class SQLExecuteQueryOperator(BaseSQLOperator):
     Executes SQL code in a specific database
     :param sql: the SQL code or string pointing to a template file to be executed (templated).
     File must have a '.sql' extensions.
+
+    When implementing a specific Operator, you can also implement `_process_output` method in the
```

Comment:
should we use double backquotes in docstrings?

Reply:
Likely :). I think it will work anyway in docstring  (we'll see in the API docs) but yeah - we can fix it as follow-up